testsuites/CBLTester/CBL_Functional_tests/APITests/test_collection.py

```diff
@@ -0,0 +1,264 @@
+import pytest
+from keywords.utils import random_string, log_info
+
+# These test cases are only for checking the framework
+
+
+@pytest.mark.usefixtures("class_init")
+class TestDatabase(object):
+
+    @pytest.mark.parametrize("db_name", [
+        random_string(1),
+        random_string(6),
+        random_string(128),
+        "_{}".format(random_string(6)),
+        "{}_".format(random_string(6)),
+        "_{}_".format(random_string(6)),
+        random_string(6, digit=True),
+        random_string(6).upper(),
+    ])
+    # Test to check default scope name in CBL.
+    def test_defaultScope(self, db_name):
+        db = self.db_obj.create(db_name)
+        scope = self.db_obj.defaultScope(db)
+        scopeName = self.scope_obj.scopeName(scope)
+        expected_name = "_default"
+        assert scopeName == expected_name, "Default scope not present"
+
+    @pytest.mark.parametrize("db_name", [
+        random_string(1),
+        random_string(6),
+        random_string(128),
+        "_{}".format(random_string(6)),
+        "{}_".format(random_string(6)),
+        "_{}_".format(random_string(6)),
+        random_string(6, digit=True),
+        random_string(6).upper(),
+    ])
+    # Test to check default collection name in CBL
+    def test_defaultCollection(self, db_name):
+        db = self.db_obj.create(db_name)
+        collection = self.db_obj.defaultCollection(db)
+        collectionName = self.collection_obj.collectionName(collection)
+        expected_name = '_default'
+        assert collectionName == expected_name, "Wrong Default Collection Name"
+
+    @pytest.mark.parametrize("db_name, collectionName", [
+        (random_string(6), random_string(6)),
+        (random_string(6), random_string(6)),
+        (random_string(6), random_string(6)),
+        (random_string(6), random_string(6)),
+        (random_string(6), random_string(6)),
+        (random_string(6), random_string(6)),
+    ])
+    def test_createCollection(self, db_name, collectionName):
+        db = self.db_obj.create(db_name)
+        scopeName = '_default'
```

Comment:
Maybe change the name to indicate that we're adding a collection to the default scope, to be more specific?
Something like: test_create_collection_to_default_scope?